vm/devices/storage/nvme/src/namespace.rs

```diff
@@ -228,6 +228,7 @@ fn map_disk_error(err: disk_backend::DiskError) -> NvmeError {
         disk_backend::DiskError::MemoryAccess(err) => {
             NvmeError::new(spec::Status::DATA_TRANSFER_ERROR, err)
         }
+        disk_backend::DiskError::Hv(_) => spec::Status::DATA_TRANSFER_ERROR.into(),
```

Comment:
Forgive my lack of Rust-idiom knowledge, but shouldn't this be ...

```
disk_backend::DiskError::Hv(_) => {
            NvmeError::new(spec::Status::DATA_TRANSFER_ERROR, err)
        }
```

Reply:
There're two ways to convert spec::Status to NvmeError: 1)through NvmeError::new() or 2) using the implicit .into(). For this specific case, NvmeError::new requires err with extra restrict which HvError does not implement so I just use the .into().

Reply:
We can implement the required traits or other approach if there's a need to keep the source.